tsl/test/sql/cagg_policy.sql

```diff
@@ -630,3 +630,18 @@ SELECT * FROM deals_best_daily ORDER BY bucket LIMIT 2;
 -- expect to get an up-to-date notice
 CALL refresh_continuous_aggregate('deals_best_weekly', '2022-04-24', '2022-05-05');
 SELECT * FROM deals_best_weekly;
+
+-- github issue 5907
```

Comment:
instead of mentioning issue number, it will be better if we mention what is being tested here exactly.

Reply:
added a line about what this test is meant to check